src/service/media/mod.rs

```diff
@@ -187,7 +187,7 @@ impl Service {
 				Ok(duration) => {
 					debug!("Parsed duration: {:?}", duration);
 					debug!("System time now: {:?}", SystemTime::now());
-					SystemTime::now() - duration
+					SystemTime::now().checked_sub(duration).unwrap()
```

Comment:
I guess the panic is only triggerable through admin commands, an error message would still be nicer

Reply:
For some reason it did not occur to me that I can do `ok_or_else`'s for these checked adds.